vyper/context/types/value/array_value.py

```diff
@@ -62,6 +62,8 @@ def length(self):
 
     @property
     def size_in_bytes(self):
+        # the first slot (32 bytes) stores the actual length, and then we reserve
+        # enough additional slots to store the data if it uses the max available length
```

Comment:
```suggestion
        # enough additional slots to store the data if it uses the max available length
        # because this data type is single-bytes, we make it so it takes the max 32 byte
        # boundary as it's size, instead of giving it a size that is not cleanly divisble by 32
```